frontend/src/components/CongratulationsPopupWindow.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useRef, useState } from "react";
+import styles from "./CongratulationsPopupWindow.module.css";
+import closeIcon from "../assets/closeIcon.svg";
+import checkMark from "../assets/checkMarkIcon.svg";
+import { Button } from "./Button.tsx";
+
+interface CongratulationsPopupWindowProps {
+  buttonText: string; // Define prop for button text
+}
+const CongratulationsPopupWindow: React.FC<CongratulationsPopupWindowProps> = ({ buttonText }) => {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
I think the state of the popup should go outside the pop up this way we can manage the state of the popup from outside. We many not always be triggering the popup using a button. Or at least a way to set the state of the popup from outside the component

Either with state logic outside for example the [twobuttonpopup here](https://github.com/TritonSE/YSC-Mobile-Application/blob/main/mobile/src/screens/LoadingScreen.tsx)

Or with a way to set the logic from outside like how [Accordian is used in FAQ](https://github.com/TritonSE/CCIDC-Application-Overhaul/blob/7-home-page/frontend/src/components/FAQs.tsx)

Reply:
Example on Candidates.tsx